### PR TITLE
Add Redmi 4X device maintainer

### PR DESCRIPTION
**Pull request by TingyiChen**

### CODE REVIEW

File: res/values/resurrection_device_maintainers_strings.xml [hunk 1]
```diff
@@ -592,6 +592,9 @@
   <string name="device_land">Xiaomi Redmi 3s</string>
   <string name="device_land_summary">land\nMaintainer - Saicharan </string>
 
+  <string name="device_santoni">Xiaomi Redmi 4X</string>
+  <string name="device_santoni_summary">santoni\nMaintainer - IsaacChen </string>
```
Comment (arvind-gr) [inline res/values/resurrection_device_maintainers_strings.xml:596]: I searched and couldn't find Official XDA page for this official dev. Is forum to be created?

Comment (TingyiChen) [reply res/values/resurrection_device_maintainers_strings.xml:596]: https://forum.xda-developers.com/android/development/rom-resurrectionremix-5-8-4-t3653793 
 thread is here

Comment (arvind-gr) [reply res/values/resurrection_device_maintainers_strings.xml:596]: Oh Thanks a lot for instant reply. Santoni's ROM dev is scattered at common Android development, Is https://forum.xda-developers.com/showpost.php?p=73396004&postcount=22180 enough or how to get Santoni subforum in XDA?..

Comment (TingyiChen) [reply res/values/resurrection_device_maintainers_strings.xml:596]: need 50+ threads

**Pull request merged**